src/main/java/org/embulk/input/kintone/KintoneInputPlugin.java

```diff
@@ -113,4 +142,94 @@ protected KintoneClient getKintoneClient()
     {
         return new KintoneClient();
     }
+
+    private List<Record> expandSubtable(final Record originalRecord, final List<String> subTableFieldCodes)
+    {
+        ArrayList<Record> records = new ArrayList<>();
+        records.add(cloneRecord(originalRecord));
+        for (String fieldCode : subTableFieldCodes) {
+            List<TableRow> tableRows = originalRecord.getSubtableFieldValue(fieldCode);
+            for (int idx = 0; idx < tableRows.size(); idx++) {
+                if (records.size() < idx + 1) {
+                    records.add(cloneRecord(originalRecord));
+                }
+
+                TableRow tableRow = tableRows.get(idx);
+                Record currentRecord = records.get(idx);
+                Set<String> tableFieldCodes = tableRow.getFieldCodes();
+                for (String tableFieldCode : tableFieldCodes) {
+                    currentRecord.putField(tableFieldCode, tableRow.getFieldValue(tableFieldCode));
+                }
+            }
+        }
+        return records;
+    }
+
+    private Record cloneRecord(final Record src)
+    {
+        Record dst = new Record(src.getId(), src.getRevision());
+        for (String fieldCode : src.getFieldCodes(true)) {
+            dst.putField(fieldCode, src.getFieldValue(fieldCode));
+        }
+        return dst;
+    }
+
+    private Schema buildSchema(final PluginTask task)
+    {
+        KintoneClient client = getKintoneClient();
+        client.validateAuth(task);
+        client.connect(task);
+
+        Map<String, FieldProperty> fields = client.getFields(task);
+        Builder builder = Schema.builder();
+
+        // built in schema
+        builder.add("$id", Types.LONG);
+
+        for (Map.Entry<String, FieldProperty> fieldEntry : fields.entrySet()) {
+            builder.add(fieldEntry.getKey(), buildType(fieldEntry.getValue().getType()));
+        }
+
+        return builder.build();
+    }
+
+    private Type buildType(final FieldType fieldType)
+    {
+        switch(fieldType) {
+            case __ID__:
+            case __REVISION__:
+            case RECORD_NUMBER:
+                return Types.LONG;
+            case CALC:
+            case NUMBER:
+                return Types.DOUBLE;
+            case CREATED_TIME:
+            case DATE:
```

Comment:
`DATE` is formatted as `%Y-%m-%d` but [`DEFAULT_TIMESTAMP_PATTERN`](https://github.com/trocco-io/embulk-input-kintone/blob/master/src/main/java/org/embulk/input/kintone/KintoneInputColumnVisitor.java#L18) is `%Y-%m-%dT%H:%M:%S%z`, so the following exception occurs.
```
2023-09-22 18:36:12.220 +0900 [WARN] (0016:task-0000): Invalid timestamp column: 日付
org.embulk.util.rubytime.RubyDateTimeParseException: Text '2023-09-02' could not be parsed at index 10
	at org.embulk.util.rubytime.ParserWithContext.consumeImmediateString(ParserWithContext.java:328) ~[na:na]
	at org.embulk.util.rubytime.ParserWithContext.parse(ParserWithContext.java:57) ~[na:na]
	at org.embulk.util.rubytime.RubyDateTimeFormatter.parseUnresolved(RubyDateTimeFormatter.java:105) ~[na:na]
	at org.embulk.util.rubytime.RubyDateTimeFormatter.parse(RubyDateTimeFormatter.java:119) ~[na:na]
	at org.embulk.util.timestamp.RubyTimestampFormatter.parse(RubyTimestampFormatter.java:43) ~[na:na]
	at org.embulk.input.kintone.KintoneInputColumnVisitor.timestampColumn(KintoneInputColumnVisitor.java:119) ~[na:na]
	at org.embulk.spi.Column.visit(Column.java:54) [embulk:0.9.26]
	at org.embulk.spi.Schema.visitColumns(Schema.java:68) [embulk:0.9.26]
	at org.embulk.input.kintone.KintoneInputPlugin.run(KintoneInputPlugin.java:111) [embulk-input-kintone-0.1.8.jar:0.1.8]
	at org.embulk.exec.LocalExecutorPlugin$ScatterExecutor.runInputTask(LocalExecutorPlugin.java:269) [embulk:0.9.26]
	at org.embulk.exec.LocalExecutorPlugin$ScatterExecutor.access$100(LocalExecutorPlugin.java:194) [embulk:0.9.26]
	at org.embulk.exec.LocalExecutorPlugin$ScatterExecutor$1.call(LocalExecutorPlugin.java:233) [embulk:0.9.26]
	at org.embulk.exec.LocalExecutorPlugin$ScatterExecutor$1.call(LocalExecutorPlugin.java:230) [embulk:0.9.26]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_382]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_382]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_382]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_382]
```

Reply:
Thank you.
I changed to treat DATE as String type.